apps/website/src/pages/settings/account.tsx

```diff
@@ -13,7 +13,7 @@ import PasswordSection from '../../components/settings/PasswordSection';
 
 const CURRENT_ROUTE = ROUTES.settingsAccount;
 
-const AccountSettingsPage = ({ auth }: { auth: { token: string } }) => {
+const AccountSettingsPage = withAuth(({ auth }) => {
```

Comment:
Wrapping using `withAuth` here, and not at the export level, means we get type inference on `auth` correctly